plop-templates/@react-spectrum/test/Component.test.js.hbs

```diff
@@ -10,19 +10,18 @@
  * governing permissions and limitations under the License.
  */
 
-import {{{componentName}}} from '../';
+import { {{~componentName~}} } from '../';
 import {render} from '@testing-library/react';
 import React from 'react';
 import V2{{componentName}} from '@react/react-spectrum/{{componentName}}';
```

Comment:
I think we're starting to get rid of the v2 tests, so i think we can remove it from the template